src/utils/hints/stateChannelsHints.js

```diff
@@ -1,9 +1,14 @@
 export const stateChannelsHints = {
   stateChannelId: 'Unique identifier of the State Channel.',
   status: 'When status is "Open" transactions can be executed inside the State Channel, outside the blockchain network.',
-  participants: 'Initiator is the account that created the State Channel via ChannelCreateTx.',
+  participants: 'Initiator is the account that created the State Channel via ChannelCreateTx. Responder is the account that joined the State Channel.',
   onChainUpdates: 'Amount of on-chain transactions within the State Channel.',
   locked: 'Currently deposited amount to the State Channel by its participants.',
   lastUpdated: 'Block height and estimated date and time of the last interaction with the State Channel.',
   lastTxType: 'Last transaction\'s type of the State Channel.',
```

Comment:
```suggestion
  lastTxType: 'Type of the last transaction in the State Channel.',
```
I know it was my part of the work, but now when I see it, I would change it like this